### PR TITLE
Test suits for home page

### PR DESCRIPTION
**Pull request by ShaileshParmar11**
### Describe your changes :

I worked on the test suits

#
### Type of change :

- [ ] Bug fix
- [x] New feature
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation

#
### Frontend Preview (Screenshots) :

#
### Checklist:

- [x] I have read the [**CONTRIBUTING**](https://docs.open-metadata.org/open-source-community/developer) document.
- [x] I have performed a self-review of my own. 
- [x] I have tagged my reviewers below.
- [ ] I have commented on my code, particularly in hard-to-understand areas.
- [x] My changes generate no new warnings.
- [x] I have added tests that prove my fix is effective or that my feature works.
- [x] All new and existing tests passed.

#
### Reviewers

Frontend: @shahsank3t, @darth-coder00, @Sachin-chaurasiya

### CODE REVIEW

File: catalog-rest-service/src/main/resources/ui/src/components/my-data-details/SchemaTable.test.tsx [hunk 1]
```diff
@@ -19,14 +19,14 @@ import { getAllByTestId, render } from '@testing-library/react';
...
 import { MemoryRouter } from 'react-router-dom';
-import { Column, ColumnDataType } from '../../generated/entity/data/table';
+import { ColumnDataType } from '../../generated/entity/data/table';
 import SchemaTable from './SchemaTable';
 
 jest.mock('../common/rich-text-editor/RichTextEditorPreviewer', () => {
   return jest.fn().mockReturnValue(<p>RichTextEditorPreviewer</p>);
 });
 
-const mockColumns: Column[] = [
```
Comment (Sachin-chaurasiya) [inline catalog-rest-service/src/main/resources/ui/src/components/my-data-details/SchemaTable.test.tsx:29]: Why did we remove the Column type?

Comment (ShaileshParmar11) [reply catalog-rest-service/src/main/resources/ui/src/components/my-data-details/SchemaTable.test.tsx:29]: its work completely fine without it so in test suit we only add things if it is required.

Comment (ShaileshParmar11) [reply catalog-rest-service/src/main/resources/ui/src/components/my-data-details/SchemaTable.test.tsx:29]: image

Comment (Sachin-chaurasiya) [reply catalog-rest-service/src/main/resources/ui/src/components/my-data-details/SchemaTable.test.tsx:29]: @ShaileshParmar11  Have you checked typescript errors as well?
when you locally run the application please check if it is working fine or not.

Comment (ShaileshParmar11) [reply catalog-rest-service/src/main/resources/ui/src/components/my-data-details/SchemaTable.test.tsx:29]: I removed it from the test file and not the original component, and everything is working fine, without any new warning/error.

**Review by Sachin-chaurasiya — APPROVED** [review]
LGTM.

**Review by darth-coder00 — APPROVED** [review]
LGTM

**Pull request merged**